src/core/lwt.mli

```diff
@@ -54,24 +54,23 @@ let () =
 
     This is all explained in the next sections:
 
-    - {{: #3_Quickstart} Quick start} links these concepts to actual functions
+    - {!section-3_Quickstart} links these concepts to actual functions
```

Comment:
All the `section-`, `type:` and `Lwt.` prefixes are not necessary, the references will resolve as long as there's no ambiguity.

Reply:
I've removed the unnecessary prefixes, but I'm keeping the module paths: it'd be too long to change at this point.